pkg/conf/schedulerconf.go

```diff
@@ -63,6 +63,7 @@ const (
 	CMSvcDisableGangScheduling  = PrefixService + "disableGangScheduling"
 	CMSvcEnableConfigHotRefresh = PrefixService + "enableConfigHotRefresh"
 	CMSvcPlaceholderImage       = PrefixService + "placeholderImage"
+	CMSvcApiClientTimeout       = PrefixService + "apiClientTimeout"
```

Comment:
Can we use "kubernetes.apiClientTimeout" instead (i.e. PrefixKubernetes). This aligns better with the existing settings.

Reply:
That makes sense. This should now be resolved. I also updated my `yunikorn-site` repo PR to reflect this change in the prefix value.